README.md

```diff
@@ -1,84 +1,83 @@
-```
-Once complete, the binary will be available at: `~/go/src/k8s.io/kubernetes/_output/dockerized/bin/linux/amd64/e2e.test`
+- https://kubernetes.io/blog/2019/03/22/kubernetes-end-to-end-testing-for-everyone/
+- https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md#building-kubernetes-and-running-the-tests
 
-#### Cross-building for Mac or Windows
+Now, assuming you are able to build an e2e.test binary, we'll proceed with how windows the tests work.
 
-To build a binary to run on Mac or Windows, you can add `KUBE_BUILD_PLATFORMS`.
+**See the Questions section at the bottom of this doc** for pointers on how to build the e2e tests. 
 
-For Windows
-```bash
-./build/run.sh make KUBE_BUILD_PLATFORMS=windows/amd64 WHAT=test/e2e/e2e.test
-```
+## Running the e2e.test binary on a windows enabled cluster
 
-For Mac
-```bash
-./build/run.sh make KUBE_BUILD_PLATFORMS=darwin/amd64 WHAT=test/e2e/e2e.test
-```
-
-Your binaries will be available at `~/go/src/k8s.io/kubernetes/_output/dockerized/bin/linux/amd64/e2e.test` where `linux/amd64/` is replaced by `KUBE_BUILD_PLATFORMS` if you are building on Mac or Windows.
-
-
-## Running an e2e test pass
+If you already have a cluster, you will likely just need to build e2e.test, and run it with windows options.
 
+### Method 1: Using an existing cluster
 
-### Using an existing cluster
+All of the tests are built into the `e2e.test` binary, which you can as a standalone binary to test an existing cluster. 
+If you don't know how to build `e2e.test`, instructions for this are at the bottom of this doc.
 
-All of the tests are built into the `e2e.test` binary, which you can as a standalone binary to test an existing cluster.
+Note that, when running these **You need to set the KUBE_TEST_REPO_LIST** environment variable when running the windows end to end tests, otherwise, windows images will not be pullable and all your tests will fail !
 
-There are a few important parameters that you need to use:
+- There are a few important parameters that you need to use, including:
 
 - `--provider=skeleton` - this will avoid using a cloud provider to provision new resources such as storage volumes or load balancers
 - `--ginkgo.focus="..."` - this regex chooses what [Ginkgo](http://onsi.github.io/ginkgo/) tests to run.
 - `--node-os-distro="windows"` - some test cases have different behavior on Linux & Windows. This tells them to test Windows.
 - `--ginkgo.skip="..."` - this regex chooses what tests to skip
 - If you're not sure what test cases will run, add `--gingkgo.dryRun=true` and it will give a list of test cases selected without actually running them.
 
-`e2e.test` also needs a few environment variables set to connect to the cluster, and choose the right test container images. Here's an example:
+- `e2e.test` also needs a few environment variables set to connect to the cluster, and choose the right test container images. Here's an example:
```

Comment:
looks like the example go moved.  This particular line might be better off in the next section

Reply:
fixed